Queue/Queue.cpp

```diff
@@ -0,0 +1,86 @@
+#include"Queue.h"
+using namespace std;
+
+void CreateQueue(Queue& q, int size)
+{
+	q.data = new int[size];
+	q.capacity = size;
+	q.front = 0;
+	q.rear = -1;
+	q.count = 0;
+}
+
+
+int size(Queue& q)
```

Comment:
Corrent the naming convention. Should be GetSize or something like that.

Reply:
I done it already